article.html

```diff
@@ -0,0 +1,359 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>New York Times Article</title>
+
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="styles.css">
+    </head>
+
+<body>
+    <header>
+        <a href="">SPACE & COSMOS</a>
+        <div>
+            <button>SUBSCRIBE NOW</button>
+            <button>LOG IN</button>
```

Comment:
- [ ] Please give proper styling and alignment. This will help you to make it like the original one.